packages/jest-resolve/src/utils.ts

```diff
@@ -98,6 +98,11 @@ export const resolveTestEnvironment = ({
   testEnvironment: string;
   requireResolveFunction: (moduleName: string) => string;
 }): string => {
+  // we don't want to resolve the actual `jsdom` env if `jest-environment-jsdom` is not installed
```

Comment:
Shouldn't it be "module"?
```suggestion
  // we don't want to resolve the actual `jsdom` module if `jest-environment-jsdom` is not installed, but `jsdom` package is
```

Reply:
right! 😅